CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 # Unreleased
 
+- **Breaking:** On Linux, the functions `is_wayland`, `is_x11`, `xlib_xconnection` and `wayland_display` have been moved to a new `EventLoopWindowTargetExtUnix` trait.
```

Comment:
I think we usually add changes to the bottom of the list, not the top.

Also, that would be the first change marked as "Breaking". Because we're in "Alpha", I didn't mark my bool->enum change breaking, and there's at least one other change in the list that would be considered breaking.

So... I think you can remove "Breaking" here, and we should decide whether we label breaking changes individually or slap the "all below are breaking" label on top again.

Reply:
Fixed. I'll push it later.